### ISSUE TITLE
Thunderbird always restores to the KDE activity it was opened on originally

### ISSUE THREAD

**Issue by katonagl** [Ximi1970/systray-x#181]
**Describe the bug**
I use activities in KDE6. After starting Tb (with the x11 workaround) and minimizing it to tray, when I switch to another activity and click the tray icon, it jumps to the original activity where Tb was opened. Previously with KDE5 it was working fine. 

**To Reproduce**
1. Setup multiple activities in KDE6.
2. Open Tb with x11 workaround so systray-x works
3. Minimize Tb to tray
4. Switch activity
5. Open from tray by clicking tray icon

**Expected behavior**
Tb should open on the current activity

**Desktop (please complete the following information):**
 - OS / distribution: openSUSE Tumbleweed
 - Architecture (32 or 64 bit): 64 bit
 - Desktop type / Window Manager (KDE, Xfce, Cinnamon, MATE etc): KDE6 with wayland
 - Thunderbird version and arch (Tb xx.xx.x, 32 or 64 bit): 115.8.1 (64 bit)
 - Thunderbird package type (repository, mozilla binary, flatpak, snap, etc): repo
 - Run `echo $XDG_SESSION_TYPE` in a terminal and report the results: wayland
 - Add-on version: 0.9.8
 - App version: 0.9.8 build 1064
 - Install type: github repository build, script installer, OBS repository: openSUSE repository  
 - Used Qt library (default system repository, other OBS repo, self-compiled, Qt online installer): openSUSE Qt6

**Additional context**
I start Tb with `Exec=env GDK_BACKEND=x11`

**Comment by katonagl** [comment]
More testing: I have activities and also multiple desktops for each activity. Restoring Tb keeps the desktop number, but on wrong activity. So if Tb is started on Activity1 and then restored when on Activity2 Desktop2, it will restore to Desktop2 of Activity1. If restored on Activity2 Desktop3, it will be restored to Activity1 Desktop3.

I have tested this also with an x11 session with the same result.

**Comment by Ximi1970** [comment]
This behavior is conform the "standard" activities. The Thunderbird is "locked" at the activity where it was started. So the systray-x icon will start TB in that activity.
If you want TB in all activities you need to tell the activity manager you want this.
This can be done by enabling the REAL titlebar of thunderbird again, Settings -> Language & Appearance -> Hide system window titlebar.
Restart TB .
Then right click on the upper left TB icon.
Select "Show in activities" and select "All activities".
You can disable the titlebar again I hope.
Maybe there is another way but I do not use this feature at all...

**Comment by Ximi1970** [comment]
Nope, you must leave the system titlebar enabled.

**Comment by katonagl** [comment]
Yes, it works. However, is it possible to change the activity of Tb by systray-x when restoring? Maybe it is not a bug then (however I remember as it was working previously, but maybe I am wrong), but a feature request then. It would be really useful if systray-x could manage the activities this way, so it does not have to appear on all activities.

**Comment by Ximi1970** [comment]
I also tested it in 15.5 but it has the behavior as in Tumbleweed. I also see the manager does not store the wanted behavior and will reset it every time you restart TB.
Maybe they changed the activities handling a bit?
I am not going to add things to repair a "broken" feature of KDE.

**Comment by Ximi1970** [comment]
You could try Left upper icon-> More actions -> Configure special application settings -> and add a activity setting?
